scripts/CMakeLists.txt

```diff
@@ -278,7 +278,7 @@ IF(WIN32)
     INSTALL_SCRIPT(${CMAKE_CURRENT_BINARY_DIR}/${file}.pl COMPONENT Server_Scripts)
   ENDFOREACH()
 ELSE()
-  IF(WITH_WSREP)
+  IF(WITH_WSREP AND NOT WITHOUT_SERVER)
```

Comment:
No need for this, please remove since in `cmake/wsrep.cmake` variable `WITH_WSREP` is controlled with `WITHOUT_SERVER`